nncf/onnx/quantization/quantize_model.py

```diff
@@ -44,6 +54,60 @@
 TTensor = TypeVar("TTensor")
 
 
+def check_model_protobuf_size(model: onnx.ModelProto) -> None:
+    """
+    Checks whether the serialized ONNX model exceeds the 2GB protobuf size limit.
+    If the size limit is exceeded, a `nncf.ValidationError` is raised.
+
+    :param model: The ONNX model to be checked.
+    """
+    MAXIMUM_PROTOBUF = 2000000000  # Limitation of single protobuf file is 2GB
+    protobuf_string = model.SerializeToString()
```

Comment:
Will a copy of the model be created by `model.SerializeToString()`?

Reply:
I guess so. There are no other solutions for resolving it (as far as I know). We can use `model.ByteSize(`) (which returns the serialized size of this message), but it also uses extra memory.